### PR TITLE
Soycode fixtests

### PR DESCRIPTION
**Pull request by soycode**
This, combined with https://github.com/willscott/webrtc-adapter/pull/6 and https://github.com/freedomjs/freedom/pull/275, addresses many of the remaining concerns in https://github.com/freedomjs/freedom-for-firefox/issues/50.

### CODE REVIEW

**Comment by soycode** [comment]
N.B. - tests will continue failing on Shippable until we cut new releases of the dependency packages (webrtc-adapter and freedom). At least, I don't know a good way to otherwise update Shippable - they do pass locally for me though.

**Comment by willscott** [comment]
should this bump the dependency versions?

**Comment by soycode** [comment]
Probably a good idea, went ahead and updated all and it still seems to run/build as it was. One annoying thing is it seems like the getStats test is failing on OSX even with the new webrtc-adapter (it passed in Linux which is where I originally wrote it). I still think this is worth merging as hopefully it keeps passing on Linux and thus Shippable.

**Comment by soycode** [comment]
And checking the Shippable results, getStats does indeed pass in Linux land, so that's good. There's still some failing TCP tests, but hopefully they pass once we also cut a fresh release of freedom.

**Comment by soycode** [comment]
At this point the only failing test is:

    >> integration: core.tcpsocket Pauses and resumes failed
    >> Error: Timeout - Async callback was not invoked within timeout specified by 
    jasmine.DEFAULT_TIMEOUT_INTERVAL. in resource://jid1-mkagayemb0e5nq-at-
    jetpack/freedom_for_firefox_tests/data/spec.jsm (line 11111)

And I think this will likely pass once we cut a new release of freedom. LMK what order you want to do things in (i.e. merging, releasing, etc.) - thanks!

File: package.json [hunk 1]
```diff
@@ -18,29 +18,29 @@
...
+    "grunt-prompt": "^1.3.0",
+    "grunt-shell": "^1.1.2",
+    "jasmine-core": "^2.3.4",
     "node-json-minify": "^0.1.3-a",
-    "webrtc-adapter": "^0.0.6"
+    "webrtc-adapter": "^0.0.7"
   },
   "peerDependencies": {
-    "freedom": "~0.6.23"
+    "freedom": "~0.6.24"
```
Comment (willscott) [inline package.json]: 25?

**Comment by willscott** [comment]
:+1:

**Pull request merged**